misc/scripts/models-as-data/bulk_generate_mad.py

```diff
@@ -341,6 +342,13 @@ def download_dca_databases(
             print(f"Skipping {pretty_name} as it is not in the list of projects")
             continue
 
+        if pretty_name in artifact_map:
+            print(f"Skipping previous database {artifact_map[pretty_name]['artifact_name']} for {pretty_name}")
```

Comment:
Would it being worth choosing the *best* database of a particular name (e.g. the most recent) or are they likely to be very close and/or difficult to compare anyway?